deps/curl-7.78.0/lib/vtls/sf_cJSON.c

```diff
@@ -0,0 +1,2938 @@
+/*
```

Comment:
@SimbaGithub are we keeping track of updates happening on the cJSON source ?  Do we have any checks and balances to ensure that the cJson fork here is in sync with the original? If not I'd suggest we come up with a plan to keep the two in sync or atleast test with the latest changes. There are not many in the past 1 year but it might be worthwhile to consider the delta.

Reply:
@sfc-gh-hchaturvedi So far we just copy sf_cJSON from older curl to the new one. I agree it's time to sync with the original because it has been 3 years since we picked up version 1.7.4 and now the original is 1.7.15. We can discuss the plan on our sync meeting. Given the fact that there are only bug fixes and performance improvements and no major changes in the past 3 years, for now I think after sync to 1.7.15, in the future we can add sync cJSON as a part of curl update task so we can do it whenever we update to a new curl version.